internal/api/courses/assignments/submissions/proxy/resubmit.go

```diff
@@ -0,0 +1,76 @@
+package proxy
+
+import (
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/db"
+	"github.com/edulinq/autograder/internal/grader"
+	"github.com/edulinq/autograder/internal/timestamp"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+type ResubmitRequest struct {
+	core.APIRequestAssignmentContext
+	core.MinCourseRoleGrader
```

Comment:
Put a newline here.
I like to separate the admin fiends from the real data fields.